data/en/1_0005_updated.json

```diff
@@ -0,0 +1,2172 @@
+                                "Long service awards",
+                                "Appearance money (sporting professionals)"
+                            ]
+                        }, {
+                            "description": "Exclude:",
+                            "list": [
+                                "Trainees on government schemes",
+                                "Directors fees",
+                                "Employer’s NI and contribution to pension schemes",
+                                "Employees working abroad unless paid directly from this business’s GB payroll",
+                                "Signing on fees (sporting professionals)",
+                                "Payment in lieu of notice",
+                                "Redundancy pay (taxable and non-taxable)",
+                                "Accrued holiday pay",
+                                "Employees in Northern Ireland",
+                                "Benefits employees receive through pay, for example, family working tax credit",
+                                "Expenses payments for attending meetings, for example, councillors"
+                            ]
+                        }]
+                    }, {
+                        "question": "Significant changes to pay or number of employees",
+                        "content": [{
+                            "description": "Include",
+                            "list": [
+                                "redundancies",
+                                "more temporary staff",
+                                "fewer temporary staff",
+                                "more overtime",
+                                "less overtime",
+                                "pay increase",
+                                "industrial action"
+                            ]
+                        }]
+                    }]
+                },
+                "secondary_content": [{
+                    "id": "how-we-use-your-data",
+                    "title": "How we use your data",
+                    "content": [{
+                        "list": [
+                            "You cannot appeal your selection. Your business was selected to give us a comprehensive view of the UK economy",
+                            "The data from your business is essential as it helps us measure the increase of wages over time"
+                        ]
+                    }]
+                }]
+            }],
+            "id": "introduction-group",
+            "title": "Introduction"
+        },
+        {
+            "blocks": [{
+                    "id": "pay-pattern-frequency",
+                    "questions": [{
+                        "answers": [{
+                            "id": "pay-pattern-frequency-answer",
+                            "label": "Please select all that apply. You will be asked to provide information for each of the selected pay patterns.",
```

Comment:
Remove "Please select all that apply." from label as this text is added by default for checkboxes now